crypto-core/src/main/java/com/palantir/crypto/io/DecryptingSeekableInput.java

```diff
@@ -96,11 +109,29 @@ public void seek(long pos) throws IOException {
         }
 
         long prevBlockOffset = prevBlock * blockSize;
-        Cipher cipher = seekableCipher.seek(prevBlockOffset);
+        CryptoCipher cipher = seekableCipher.seek(prevBlockOffset);
         delegate.seek(prevBlockOffset);
 
+        // TODO(srand) This is duplicating work already done in AesCtrCipher#seek as a result of the public
```

Comment:
I believe we should do that before merging this PR. maybe submit a pure refactor PR that sets up the signatures you need.